drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties

```diff
@@ -54,35 +54,35 @@ AdditionalInfoPageView.UpdateEngine=Update engine with changes:
+AttributeColumnPageView.DialectDescription=A string identifying either `JAVA` or `MVEL` as the language to be used for code expressions in the rule. By default, the rule uses the dialect specified at the package level. Any dialect specified here overrides the package dialect setting for the rule. Example: `dialect "JAVA"`
 AttributeColumnPageView.Duration=duration
-AttributeColumnPageView.DurationDescription=The duration dictates that the rule will fire after a specified duration, if it is still true.
+AttributeColumnPageView.DurationDescription=A long integer value defining the duration of time in milliseconds after which the rule can be activated, if the rule conditions are still met. Example: `duration 10000`
 AttributeColumnPageView.Enabled=enabled
-AttributeColumnPageView.EnabledDescription=Enabled attribute.
+AttributeColumnPageView.EnabledDescription=A Boolean value. When the option is selected, the rule is enabled. When the option is not selected, the rule is disabled.
 AttributeColumnPageView.LockOnActive=lock-on-active
-AttributeColumnPageView.LockOnActiveDescription=Whenever a ruleflow-group becomes active or an agenda-group receives the focus, any rule within that group that has lock-on-active set to true will not be activated any more; irrespective of the origin of the update, the activation of a matching rule is discarded. This is a stronger version of no-loop, because the change could now be caused not only by the rule itself. It's ideal for calculation rules where you have a number of rules that modify a fact and you don't want any rule re-matching and firing again. Only when the ruleflow-group is no longer active or the agenda-group loses the focus those rules with lock-on-active set to true become eligible again for their activations to be placed onto the agenda.
+AttributeColumnPageView.LockOnActiveDescription=A Boolean value, applicable only to rules within rule flow groups or agenda groups. When the option is selected, the next time the ruleflow group for the rule becomes active or the agenda group for the rule receives a focus, the rule cannot be activated again until the ruleflow group is no longer active or the agenda group loses the focus. This is a stronger version of the `no-loop` attribute, because the activation of a matching rule is discarded regardless of the origin of the update (not only by the rule itself). This attribute is ideal for calculation rules where you have a number of rules that modify a fact and you do not want any rule re-matching and firing again.
 AttributeColumnPageView.Negate=negate
-AttributeColumnPageView.NegateDescription=Negate attribute.
+AttributeColumnPageView.NegateDescription=A Boolean value. When the option is selected, the rule is negated.
```

Comment:
@manstis or anyone else, could you please validate my "Negate" description? It's the only one not on my existing list and I redefined it on the fly as I did it. Want to make sure it's accurate.

Reply:
The condition/when part of the rule is negated. having this attribute column checked to true will cause wrapping the original condition into **not()**. For example **not( $applicant : Applicant(age > 18) )**